rules/coding-style/src/Rector/Throw_/AnnotateThrowablesRector.php

```diff
@@ -173,83 +211,210 @@ private function identifyThrownThrowablesInStaticCall(StaticCall $staticCall): a
         if (! $thrownClass instanceof FullyQualified) {
             throw new ShouldNotHappenException();
         }
-        $classFqn = implode('\\', $thrownClass->parts);
-        $methodNode = $thrownClass->getAttribute('nextNode');
-        $methodName = $methodNode->name;
 
-        return $this->extractMethodReturnsFromDocblock($classFqn, $methodName);
+        return $this->extractMethodReturns($thrownClass, $thrownClass->getAttribute('nextNode'));
     }
 
-    private function extractMethodReturnsFromDocblock(string $classFqn, string $methodName): array
+    private function extractMethodReturns(FullyQualified $fullyQualified, $methodNode): array
```

Comment:
`$methodNode` is missing type declaration

Reply:
Yes, I forgot it. Added.